backend/codesirius/internal_api/admin.py

```
@@ -0,0 +1,144 @@
+        """Auto-assign created_by, generate API key only when creating, and show raw key once."""
+        if not obj.pk:  # If creating a new object
+            obj.created_by = request.user
+            raw_key, hashed_key = APIKey.generate_key()
+            obj.key = hashed_key
+
+            # Save the object first
+            super().save_model(request, obj, form, change)
+
+            # Clear any default messages
+            storage = messages.get_messages(request)
+            for message in storage:
```

Ruff (0.8.2)

Loop control variable `message` not used within loop body

Rename unused `message` to `_message`

(B007)